packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1192,7 +1188,7 @@ class SelectionRect {
 ///  * [TextInputConfiguration], to opt-in to receive [TextEditingDelta]'s from
 ///    the platforms [TextInput] you must set [TextInputConfiguration.enableDeltaModel]
 ///    to true.
-abstract class DeltaTextInputClient extends TextInputClient {
+mixin DeltaTextInputClient implements TextInputClient {
```

Comment:
Does it make sense to provide a default implementation for `void updateEditingValueWithDeltas(List<TextEditingDelta> textEditingDeltas);`, since this implements `TextInputClient` thus has access to `updateEditingValue`?

Reply:
I think maybe not? Since that would need to set the `TextInputClient`'s current text editing value which I don't believe we access to here. Or maybe i'm misunderstanding how the default implementation would look.

Reply:
There is. https://master-api.flutter.dev/flutter/services/TextInputClient/currentTextEditingValue.html

We can even deprecate `DeltaTextInputClient` if we move the update method to `TextInputClient` and provide a default implementation.

Reply:
This could open up opportunities for `EditableText` to use deltas since it would only require the `TextInputConfiguration` to set `enableDeltaModel` to true, maybe through a flag thats piped through `TextField`?. Another option could also be that `EditableText` mixins both `TextInputClient` and `DeltaTextInputClient`. I'm leaning towards the former since deltas seem like a core function of text input. Though the latter would avoid any breakages.

Reply:
That's exciting, I'll move forward with this PR and we can go from there in another PR.  Deltas would be pretty core especially if we were using it in EditableText.